.env.example

```diff
@@ -16,4 +16,7 @@ PORT=50002
 ELECTRUM_X_PROTOCOL_VERSION=2.0
 
 # true, false
-IS_BETA=false
\ No newline at end of file
+IS_BETA=false
+
+# http://explorer.bitcoinvault.global (mainnet) http://18.196.199.105 (testnet)
+EXPLORER_URL=http://explorer.bitcoinvault.global
```

Comment:
I'd add missing empty lines at end of files

Reply:
Also, `https` here imo

Reply:
done